WordPress/Classes/ViewRelated/NUX/SiteCreation.storyboard

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure where this is happening, so I'll just put it here. The top border on the table looks to be a bit below the actual table top (i.e there's a white line above the border):
<img width="538" alt="screen shot 2017-12-19 at 2 43 35 pm" src="https://user-images.githubusercontent.com/1816888/34180190-0969f9e0-e4cb-11e7-90de-f18ff81e839c.png">

Reply:
No clue what's going on there…